### DISCUSSION TITLE
Scope of pruning when using multiple profiles on same machine for same repo?

### DISCUSSION THREAD

**Discussion by rxgh99** [borgbase/vorta#1542] [FAQ] [answered]
Hi, 

I'm wondering what the scope of pruning is when using multiple profiles against the same repo. 

Context: I have two data locations on my laptop that I backup with two separate profiles, to support different schedules of backup. All data goes to a single repo. Each profile uses its own naming convention for the archive logs to indicate which data location is being backed up. 

Question: When I select "prune" in a profile, does that only prune the archives for that particular profile and set of sources, or does it prune the entire repo? 

I'm hoping it is profile-specific sources pruning :) 

Thanks for any insights/confirmations.

**Comment by real-yfprojects** [comment, marked as answer]
The pruning is applied to the complete repository. AFAIK vorta will also list you all the archives in a repository no matter which profile created them. However, vorta allows you to configure a prefix the names of pruned archives have to begin with. You can use this to only prune archives created by a specific profile.

**Reply by rxgh99** [reply]
Thank you @real-yfprojects 

Just to confirm that I understand -- the "Prune Prefix" option in Vorta will constrain the scope of any prune action to only those archives with the defined prefix, correct? If so, thank you - good to understand.

**Reply by real-yfprojects** [reply]
> Just to confirm that I understand -- the "Prune Prefix" option in Vorta will constrain the scope of any prune action to only those archives with the defined prefix, correct? If so, thank you - good to understand.

Yes, it does.

**Reply by rxgh99** [reply]
Thank you for confirming.
